src/components/TenantSelector/TenantSelector.tsx

```diff
@@ -117,7 +121,7 @@ class TenantSelector extends Component<
           </SubTitle>
         )}
         <Form onSubmit={this.checkTenantValidity}>
-          <Form.Item hasFeedback={true} {...formItemProps}>
+          <Form.Item {...inputBehaviorOptions} {...formItemProps}>
```

Comment:
You can put it inside single curly brackets, like:
```jsx
<Form.Item {...inputBehaviorOptions, ...formItemProps}>
```

Reply:
[I don't think it works?](https://stackoverflow.com/questions/49470037/how-to-pass-several-props-in-spread-operator-in-react)

Reply:
yep, you're right, then I'm ok with your solution